pynecone/utils.py

```diff
@@ -799,17 +799,31 @@ def run_backend_prod(
     setup_backend()
 
     num_workers = get_num_workers()
-    command = constants.RUN_BACKEND_PROD + [
-        "--bind",
-        f"0.0.0.0:{port}",
-        "--workers",
-        str(num_workers),
-        "--threads",
-        str(num_workers),
-        "--log-level",
-        str(loglevel),
-        f"{app_name}:{constants.APP_VAR}()",
-    ]
+    command = []
+    if platform.system() == "Windows":
+        command += constants.RUN_BACKEND_PROD_WINDOWS + [
+            "--host",
+            "0.0.0.0",
+            "--port",
+            str(port),
+            "--workers",
+            str(num_workers),
+            "--log-level",
+            loglevel,
+            f"{app_name}:{constants.APP_VAR}",
+        ]
+    else:
+        command += constants.RUN_BACKEND_PROD + [
```

Comment:
It seems like all the args we pass in are the same. Can we then change this to just
```python
command = [constants..RUN_BACKEND_PROD_WINDOWS] if platform.system() == "Windows" else [constants.RUN_BACKEND_PROD]
command += [
    # Common flags here
]

```
If there are some special flags, we can put them in the if/else but move the common ones out.

Reply:
OK. Only workers arg is the same, others are little bit different.
I will make common arg list.